index.bs

```diff
@@ -270,19 +274,36 @@ the user agent <em>must</em> run the following steps:
 When <dfn>parsing an animator timeline list</dfn> for {{animatorCtor}},
 the user agent <em>must</em> run the following steps:
 
-    1. Let |properties| be an empty <code>sequence&lt;DOMString></code>.
+    1. Let |timelines| be an empty <code>sequence&lt;Object></code>.
 
     2. Let |timelinesIterable| be the result of <a>Get</a>(|animatorCtor|, |timelines|).
 
     3. For each |timeline| in |timelinesIterable|, run the following steps:
 
-        - If |timeline| is not "document" or "scroll", <a>throw</a> a <a>NotSupportedError</a>
-            and abort all these steps.
+        1. If <a>Type</a>(|timeline|) is {{DOMString}}, treat |timeline| as if it had been specified
+             as <code>{ type: |timeline|, options: {} }</code>.
```

Comment:
I think this might be better codified as:
If Type(|timeline|) is DOMString, let timelineObject be {type: |timeline|, options: {}}, otherwise let |timelineObject| be |timeline|.